pydm/widgets/tab_bar.py

```diff
@@ -11,11 +11,13 @@ def __init__(self, parent=None):
         super(PyDMTabBar, self).__init__(parent=parent)
         self.tab_channels = {}
         self._channels = None
-        self.alarm_icons = (IconFont().icon('circle', color=self.qcolor_for_alarm(self.ALARM_NONE, alarm_type=self.ALARM_INDICATOR)), 
-                        IconFont().icon('circle', color=self.qcolor_for_alarm(self.ALARM_MINOR, alarm_type=self.ALARM_INDICATOR)),
-                        IconFont().icon('exclamation-circle', color=self.qcolor_for_alarm(self.ALARM_MAJOR, alarm_type=self.ALARM_INDICATOR)),
-                        IconFont().icon('question-circle', color=self.qcolor_for_alarm(self.ALARM_INVALID, alarm_type=self.ALARM_INDICATOR)),
-                        IconFont().icon('times-circle', color=self.qcolor_for_alarm(self.ALARM_DISCONNECTED, alarm_type=self.ALARM_INDICATOR)))
+        self.alarm_icons = (
```

Comment:
This comment is just a note for myself to investigate how the alarm icon colors work under this PR.  My guess is that they don't work at all.

Reply:
It will not work as is...
One suggestion is to add two properties to the Tab Bar... iconName and iconColor...
This way you can tweak both properties using the QSS and make it work from there! 👍